aptos-move/framework/supra-framework/sources/aptos_governance.spec.move

```diff
@@ -1,4 +1,4 @@
-spec supra_framework::aptos_governance {
```

Comment:
@sjoshisupra We should also rename the file.

Reply:
done